roles/openshift_cnf/README.md

```diff
@@ -27,30 +27,53 @@ email_address                 | "mail@example.com"                   | String. E
 
 ## Example Configuration of Openshift-cnf certification project creation
 ```yaml
----
-# Generic DCI config
-dci_topic: OCP-4.15
-dci_name: Testing Openshift-cnf auto creation and attach
-dci_configuration: Using DCI create cnf project and attach product-list
-dci_config_dirs: [/etc/dci-openshift-agent]
-dci_gits_to_components: []
-
-# Certification config
-partner_creds: "/var/lib/dci-openshift-app-agent/auth.json"
-pyxis_apikey_path: "/var/lib/dci-openshift-app-agent/pyxis-apikey.txt"
-organization_id: 12345678
-check_for_existing_projects: true
-
-# cnf_name is a free-text formatted as following:
-# CNF-version + OCP-version e.g "CNF23.5 OCP4.12.9"
-cnf_to_certify:
-  - cnf_name: "test-smf23.5 OCP4.11.5"
-    pyxis_product_lists:
-      - "xxxxxxxxxxxxxxxxxxxxxxxx"
-      - "yyyyyyyyyyyyyyyyyyyyyyyy"
-  - cnf_name: "test-upf23.5 OCP4.11.5"
-
-cert_settings:
-  email_address: "email@example.com"
+# Generic DCI Pipeline config
+- name: create-openshift-cnf
+  stage: workload
+  topic: OCP-4.16
+  ansible_playbook: /usr/share/dci-openshift-app-agent/dci-openshift-app-agent.yml
+  ansible_cfg: ~/my-dci-pipeline-test/pipelines/ansible.cfg
```

Comment:
I know it's just an example, but to be more generic, can we replace `~` with something like `/var/lib/dci-openshift-app-agent` here and the variables below, thanks

Reply:
Ack. Thanks, Manny!